services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function uses `MovieSession.objects.get`, which will raise a `MovieSession.DoesNotExist` exception if the session with the given ID does not exist. Consider handling this exception to provide a more user-friendly response.